django/views/templates/default_urlconf.html

```diff
@@ -2,251 +2,244 @@
+     }
+     @keyframes smoke {
+       0% {
+         transform: translate3d(-5px, 0, 0);
+       }
+       100% {
+         transform: translate3d(5px, 0, 0);
+       }
+     }
+     .flame {
+       animation: burnInner2 .1s 70 ease-in-out alternate;
+     }
+     @keyframes burnInner2 {
+       0% {
+         transform: translate3d(0, 0, 0);
+       }
+       100% {
+         transform: translate3d(0, 3px, 0);
+       }
+     }
+     @keyframes thrust {
+       0% {
+         opacity: 1;
+       }
+       100% {
+         opacity: .5;
+       }
+     }
+     @media (prefers-reduced-motion: reduce) {
+       .exhaust__line,
+       .smoke,
+       .flame {
+         animation: none;
+       }
+     }
+     h1 {
+       font-size: 1.375rem;
+       max-width: 32rem;
+       margin: 5px auto 0;
+     }
+     main p {
+       line-height: 1.25;
+       max-width: 26rem;
+       margin: 15px auto 0;
+     }
+     footer {
+       display: grid;
+       grid-template-columns: 1fr 1fr 1fr;
+       gap: 5px;
+       padding: 25px 0;
+       position: fixed;
+       box-sizing: border-box;
+       left: 50%;
+       bottom: 0;
+       width: 960px;
+       transform: translateX(-50%);
+       transform-style: preserve-3d;
+       border-top: 1px solid #efefef;
+     }
+     .option {
+       display: grid;
+       grid-template-columns: min-content 1fr;
+       gap: 10px;
+       box-sizing: border-box;
+       text-decoration: none;
+     }
+     .option svg {
+       width: 1.5rem;
+       height: 1.5rem;
+       fill: gray;
+       border: 1px solid #d6d6d6;
+       padding: 5px;
+       border-radius: 100%;
+     }
+     .option p {
+       font-weight: 300;
+       line-height: 1.25;
+       color: #525252;
+       display: table;
+     }
+     .option .option__heading {
+       color: #092e20;
+       font-size: 1.25rem;
+       font-weight: 400;
+     }
+     @media (max-width: 996px) {
+       body, footer {
+         max-width: 780px;
+       }
+     }
+     @media (max-width: 800px) {
+       footer {
+         height: 100%;
+         grid-template-columns: 1fr;
+         gap: 60px;
+         position: relative;
+         padding: 25px;
+       }
+       .figure {
+         margin-top: 10px;
+       }
+       main {
+         padding: 0 25px;
+       }
+       main h1 {
+         font-size: 1.25rem;
+       }
+       footer {
+         width: 100%;
+         margin-top: 50px;
+       }
+     }
+     @media (min-width: 801px) and (max-height: 730px) {
+       .figure {
+         margin-top: 80px;
+       }
+     }
+     @media (min-width: 801px) and (max-height: 600px) {
+       footer {
+         position: relative;
+         margin: 135px auto 0;
+       }
+       .figure {
+         margin-top: 50px;
+       }
+     }
+     .sr-only {
+       clip: rect(1px, 1px, 1px, 1px);
+       clip-path: inset(50%);
+       height: 1px;
+       overflow: hidden;
+       position: absolute;
+       white-space: nowrap;
+       width: 1px;
+     }
```

Comment:
This is now a 1-space indent. Please make it consistent.

Reply:
Sorry, but I honestly don't know what you mean by that. As far as I can see there is 2-space indent everywhere. You mean this?

![image](https://github.com/django/django/assets/1659148/67c52e32-81a9-4581-a54e-9a7ca9cea6ef)

This is also a 2-space indent, but could be misleading because of the first character being a dot `.` 

But maybe I'm missing something here?

Reply:
I'm talking about the entire `<style>`:
```
    <style>
     html {
       line-height: 1.15;
     }
    ...
```

Reply:
Ah, yes, sorry. After a while you don't see the spaces anymore... 🤣 

I missed setting one config in the formatter to 2.